WORKSPACE

```diff
@@ -106,3 +106,13 @@ new_http_archive(
     type = "zip",
     url = "https://github.com/nunit/nunitv2/releases/download/2.6.4/NUnit-2.6.4.zip",
 )
+
+maven_jar(
+    name = "scala-xml",
+    artifact = "org.scala-lang.modules:scala-xml_2.11:1.0.5",
+)
+
+maven_jar(
+    name = "scalatest",
+    artifact = "org.scalatest:scalatest_2.11:2.2.4",
```

Comment:
I looked at the dependencies here:

http://central.maven.org/maven2/org/scalatest/scalatest_2.11/2.2.4/scalatest_2.11-2.2.4.pom

It looks like everything other than scala is optional or test. This does mean using generator/scalacheck style won't work here, but I think that's okay for a first draft.